### PR TITLE
add genomiser publication

### PR DESCRIPTION
**Pull request by visze**
* add genomiser publication
* replace wrong URL for ressources
* add ReMM-Score
* Add some posts!

### CODE REVIEW

File: _posts/2016-10-03_remm_score.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: ReMM score
+---
+
+Due to many request about the regulatory Mendelian mutation score (ReMM score) we generated a small webpage including te main sources. You can find teh ReMM score website [here](https://charite.github.io/software-remm-score.html).
```
Comment (julesjacobsen) [inline _posts/2016-10-03_remm_score.md]: Typos! te , teh -> the

**Comment by visze** [comment]
Sure.

This post is not online. So it doesn't matter. But I have no idea why
..

Am 05.10.2016 12:11 schrieb "Jules Jacobsen" <notifications@github.com>:

**Comment by julesjacobsen** [comment]
Looks like it came in as a pull request and needs merging to make it active?





From: Max
Sent: ‎Wednesday‎, ‎October‎ ‎5‎, ‎2016 ‎11‎:‎13‎ ‎AM
To: exomiser/Exomiser
Cc: Jules Jacobsen, Comment




Sure.

This post is not online. So it doesn't matter. But I have no idea why
..

Am 05.10.2016 12:11 schrieb "Jules Jacobsen" <notifications@github.com>:

**Comment by visze** [comment]
I merged the pull request already ;-)

Am 05.10.2016 12:15 schrieb "Jules Jacobsen" <notifications@github.com>:

**Pull request merged**